app/helpers.php

```diff
@@ -137,3 +137,10 @@ function has_include($request, $include)
 
     return false;
 }
+
+function formatDateAttribute($value, $timezone)
```

Comment:
Nitpicky, but Laravel's convention is to use `snake_case` for helper functions. I'm also wondering if this would be a bit clearer named as `to_date` (since it's more making a "conversion" rather than formatting).

Reply:
would it be too much to say `convert_to_date`? i'm all for making it more clear!

Reply:
That sounds great!